src/index.html

```diff
@@ -11,7 +11,166 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <main>
+      <form
+        action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+        method="post">
+        <fieldset class="fieldInput">
+          <legend class="title1">Personal information</legend>
+          <label class="unit">
+            Surname:
+            <input
+              type="text"
+              id="surname"
```

Comment:
you've wrapped your `<input>` into a `<label>`, so in this way this `id` is useless. Check for other cases as well